src/client.ts

```diff
@@ -4045,4 +4046,27 @@ export class StreamChat<StreamChatGenerics extends ExtendableGenerics = DefaultG
       ...options,
     });
   }
+
+  /**
+   * queryDrafts - Queries drafts for the current user
+   *
+   * @param {object} [options] Query options
+   * @param {object} [options.filter] Filters for the query
+   * @param {number} [options.limit] Limit the number of results
+   * @param {string} [options.next] Pagination parameter
+   *
+   * @return {Promise<APIResponse & { drafts: DraftResponse<StreamChatGenerics>[]; next?: string }>} Response containing the drafts
+   */
+  async queryDrafts(
+    options: {
+      filter?: Record<string, unknown>;
```

Comment:
Here we should be able to define what fields are permitted to query by as we do in other filters.